integreat_cms/static/src/css/style.scss

```
@@ -227,14 +227,22 @@ select,
 [type="radio"] {
     @apply rounded text-blue-500 mr-2 align-text-top;
 
-    &:disabled {
+    &:disabled,
+    .fake-disable {
```

```suggestion
    &.fake-disable {
```
I think we need the & to get this to work